assets/js/theme/cart.js

```diff
@@ -98,7 +98,7 @@ export default class Cart extends PageManager {
             const $messageBox = $('.alertMessageBox');
             const item = $('[name="item_id"]', $form).attr('value');
 
-            utils.api.productAttributes.optionChange(item, $form.serialize(), (err, result) => {
```

Comment:
looking at the new `optionChange` api, it seems that we don't need to pass `null` here